config/crd/bases/keda.sh_triggerauthentications.yaml

```diff
@@ -115,7 +115,6 @@ spec:
                   vaultUri:
                     type: string
                 required:
-                - credentials
```

Comment:
Is this intentional?

Reply:
Yes, because if they're using pod identity / workload identity the key vault credentials are no longer required. So they shouldn't be mandatory.